Kevin's local build directory being shown?

It appears that the folder on @kevinmcconnell 's machine where the app was built is being output when the go app errors out:

```sh
$ bundle exec thrust ./app.rb
{"time":"2024-04-28T12:25:33.26878-07:00","level":"INFO","msg":"Server started","http":":80"}
{"time":"2024-04-28T12:25:33.269463-07:00","level":"INFO","msg":"Server stopping"}
{"time":"2024-04-28T12:25:33.26952-07:00","level":"INFO","msg":"Server stopped"}
panic: fork/exec ./app.rb: permission denied

goroutine 1 [running]:
github.com/basecamp/thruster/internal.(*Service).Run(0x1400003ff20)
        /Users/kevin/Work/basecamp/thruster/internal/service.go:40 +0x2a4
main.main()                                                                                                                     /Users/kevin/Work/basecamp/thruster/cmd/thrust/main.go:25 +0x94```

In my case, `app.rb` is not executable and I should have used `thrust ruby ./app.rb`.  I did find it rather worrying to see Kevin's home directory when I trued to run the command, though.

Not swish enough with Go to suggest a fix here.  But if anyone has suggestions of what to Google for, I'd love to contribute a PR.

--- a/Makefile
+++ b/Makefile
@@ -9,7 +9,7 @@ build:
 dist:
 	@for platform in $(PLATFORMS); do \
 		for arch in $(ARCHITECTURES); do \
-			GOOS=$$platform GOARCH=$$arch go build -o dist/thrust-$$platform-$$arch ./cmd/...; \
+			GOOS=$$platform GOARCH=$$arch go build -trimpath -o dist/thrust-$$platform-$$arch ./cmd/...; \
 		done \
 	done
 

